apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,68 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  // Demo saveConclusion function
+  const saveConclusion = () => {
+    // Demo getRegistryAnswer
+    const getRegistryAnswer = (id) => {
+      // How do we get the registry answer?
+      return "true";
+    };
+
+    // Demo getQuestionAndAnswers
+    const getQuestionAndAnswers = () => {
+      return this.stack.reduce((acc, question) => {
+        acc.push({
+          ...question,
+          // id: does `question` contain `id`? (Yes)
+          // text: does `question` contain `text`?
+          // answer: does `question` contain `answer`? // key: `answer` or `userAnswer`? (Yes)
+          // message: why do we need `message` again?
+          // questionHash: why do we need `questionHash` again?
+          registryAnswer: getRegistryAnswer(question.id), // key: `registryAnswer` or `registerAnswer`
+        });
+        return acc;
+      }, []);
+    };
+
+    const mutationData = {
+      environment: "test/acc/prod", // Send from client?
+      date: new Date(), // Send from client?
+
+      environmentUrl: window.location.href,
+      topic: topic,
+      checkVersion: "1.0", // getCheckVerion()
+      // Key: `checkVersion` or `checkerVersion` or just `version`
+
+      questionsAndAnswers: getQuestionAndAnswers(),
+      // Example array:
+      // questionsAndAnswers: [{
+      //   id: '2637r87fuw-example-hash',
+      //   text: 'Question 1 title',
+      //   answer: 'false',
+      //   registryAnswer: 'true',
+      //   questionHash: '6273q87yd3ije-example-hash',
+      // }]
+
+      conclusions: conclusions,
+      // Key name `conclusions` or `permits`?
+      // Do we need more / specific data?
+      // Example array:
+      // permits: [{
+      //   title: "Conclusion title 1",
+      //   description: "Conclusion description 1",
+      // }]
+
+      // Only store `address` when user agrees to save personal data and wants to share the conclusion
+      address: {
```

Comment:
Maybe we should split this into user provided data and resolved data from API. That way we can check later if the address was actually correctly resolved by GraphQL.

Maybe something like this:

```
address: {
  input: {
    postalCode,
    houseNumber,
    houseNumberAddition,
  },
  resolved: {
    streetName,
    houseNumberFull,
  }
}
```
Could also be 2 separate fields, `addressInput`, `addressLookup`.

So yeah, I think it's good to make a separation between the 2.

Reply:
I'm going to use what we discussed today:

```
userInput: {
  address: {
    postalCode: '',
    houseNumberFull: '',
  },
  // geo: {
    // ...
  // },
},
registerLookups: {
  monument: { },
  cityScape: { },

  // Only store BAG if user allows
  bag: { streetname: '' },
},
```